### PR TITLE
Fixes after #161

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -78,15 +83,13 @@ if (CMAKE_VERSION VERSION_LESS 3.1 OR NOT USE_COMPILE_FEATURES)
...
-set (CMAKE_CXX_FLAGS "-O3 -g -mavx -mfma ${CMAKE_CXX_FLAGS}")
-
 if (BUILD_PYTHON)
   find_package (PythonInterp)
   find_package (PythonLibs)
   find_package (NumPy)
 
   # python site dir
-  if (PYTHON_EXECUTABLE AND NOT DEFINED PYTHON_SITE_DIR)
+  if (PYTHON_EXECUTABLE AND NOT DEFINED PYTHON_SITE_PACKAGES)
```
Comment (costashatz) [inline CMakeLists.txt]: Why are you changing the PYTHON_EXECUTABLE? #161 did not touch these lines..

Comment (jschueller) [reply CMakeLists.txt]: just a rename, I should have set this in a separate commit

**Comment by beniz** [comment]
The Unix format commit obfuscates the changes, can we have that separately maybe ? Thanks @jschueller for spotting our misdoings after a hasty merge.

**Comment by jschueller** [comment]
@beniz it's already in a spearate commit that only contains line ending change

**Comment by jschueller** [comment]
@beniz could you enable appveyor builds ?

**Comment by beniz** [comment]
Sorry I meant different PR, but it's not critical. We have Travis already, is it missing something you need ?

**Comment by jschueller** [comment]
Yes, you could set it up to build on windows so as to avoid this kind of breakage.

**Comment by jschueller** [comment]
@beniz is this ok to go ?

**Comment by beniz** [comment]
Thanks, I'll check the builds shortly. There can't be direct support for Windows at the moment, but if you PR the tools for Windows builds I'll merge them as needed.

**Pull request merged**